server/storeAppointment.php

```diff
@@ -105,13 +105,13 @@ function storeAppointment($data){
 
 		// if email is set and passes simple validation (x@x)
 		if($data['email'] && preg_match('/.+@.+/', $data['email'])){
-			mail($data['email'], 'Lincoln VITA - Appointment', $response['message']);
+			// mail($data['email'], 'Lincoln VITA - Appointment', $response['message']);
 		}
 	} catch (Exception $e) {
 		$DB_CONN->rollback();
 
 		// TODO
-		mail('someoneimportant@important.com', 'Please help, everything is on fire?', print_r($e, true).print_r($data, true));
+		// mail('someoneimportant@important.com', 'Please help, everything is on fire?', print_r($e, true).print_r($data, true));
```

Comment:
I think we can use our new vita email for this: vita@cse.unl.edu. Thoughts?

Reply:
As mentioned in our phone call, I think that it's a good idea, but like you said it would be nice to have our main emails cc'd to also sent to, because chances are we won't be checking the vita email all the time. (Although I suppose we could set up like a POP3 or whatever and have it forwarded to us, then we wouldn't have to repeat a line of code). Perhaps a forwarding system is the way to go...